### PR TITLE
[AIRFLOW-3596] Clean up undefined template variables.

### PR DESCRIPTION
**Pull request by jmcarp**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3596
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by feng-tao** [comment]
what is your approach to identify these are unused variables?

**Comment by jmcarp** [comment]
@feng-tao: I configured jinja to raise exceptions for undefined template variables using `jinja2.StrictUndefined` and ran view tests to find templates with undefined variables. Then I either passed those undefined variables to the templates if they appeared to be used, deleted them from templates if no views *ever* passed those variables, or explicitly tested that the variables were defined using the `is defined` check in jinja2.

Once https://github.com/dpgaspar/Flask-AppBuilder/pull/883 is merged, we should be able to set `StrictUndefined` and avoid undefined template variables in the future.

**Comment by feng-tao** [comment]
thanks for the info.PTAL other committers (@ashb, @bolkedebruin , @kaxil , @Fokko ) and see if they have any questions.

File: airflow/www/views.py [hunk 1]
```diff
@@ -681,9 +682,17 @@ def dag_details(self, session=None):
             .group_by(TI.state)\
             .all()
 
+        active_runs = models.DagRun.find(
+            dag_id=dag.dag_id,
+            state=State.RUNNING,
```
Comment (feng-tao) [inline airflow/www/views.py:687]: thanks. I go back and check https://github.com/apache/airflow/blob/master/airflow/models/__init__.py#L3560 which we define active_run to only running state. Initially, I thought scheduled would be one of them as well.

**Comment by feng-tao** [comment]
LGTM. assuming no comment from others. Merge now. Thanks @jmcarp .

**Pull request merged**